modules/menuItems.js

```diff
@@ -432,6 +432,32 @@ var menuTempl = function(webviews) {
         ]
     })
 
+    let languageMenu =
+    Object.keys(i18n.options.resources).map(lang_code => {
+        menuItem = {
+            label: lang_code,
+            click: function(){
+                let mainWindow = Windows.getByType('main');
+                mainWindow.webContents.executeJavaScript(
+                    `TAPi18n.setLanguage("${lang_code}");`
```

Comment:
Since `ipcMain` is an `eventsEmitter` class, the event can be triggered via `emit` function from `ipc` instance. e.g.
```
ipc.emit('backendAction_setLanguage', {}, lang_code);
```
see [doc](https://nodejs.org/api/events.html#events_emitter_emit_eventname_arg1_arg2) for more detail.